src/olympia/devhub/templates/devhub/new-landing/components/navigation.html

```diff
@@ -1,25 +1,27 @@
-<nav class="DevHub-Navigation {% if user_authenticated %}DevHub-Navigation--signedin{% endif %}">
-  <ul>
-    <li>
-      <a href="#" class="Logo {% if user_authenticated %}Logo--secondary{% endif %}">
-        <span>{{ _('Add-on Developer Hub') }}</span>
-      </a>
-    </li>
+<nav class="DevHub-Navigation content {% if user_authenticated %}scheme-light{% else %}scheme-medium{% endif %}">
```

Comment:
You could use a mixin. Again the resulting repetition might seem counter-intuitive but I remember seeing a talk in the past that being DRY via mixins will result in repetition but it compresses well it's not necessarily worth worrying about too much.